swap_meet/item.py

```diff
@@ -1,2 +1,22 @@
-class Item:
-    pass
+class Item():
+
+    def __init__(self, category = "", condition = None, age = None):
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        if self.condition in range(0,1):
```

Comment:
I suspect that `range` here isn't doing what you intend.  The `range` function returns an iterable structure similar to a list with integers from the first parameter up to but not including the second.  The condition can be a float, so instead of testing if the condition is between 0 and 1, this will test if the condition is in a list-like structure that contains only 0.